cran-comments.md

```diff
@@ -1,5 +1,36 @@
 # CRAN Submission History
 
+## v 0.4.0
+
+### Submission on April 1st, 2019
+This is a minor release that adds vignette creation functionality and makes a number of other minor changes and bug fixes.  Please see `NEWS.md` for details.
+
+As always, issues are tracked and source code is maintained at https://github.com/UptakeOpenSource/pkgnet.
+
+### CRAN Response on April 2nd, 2019
+Thanks, we see:
+
+   Found the following (possibly) invalid URLs:
+   ...
+   [two in the CreatePackageVignette()]
+   [one in NEWS.md]
+   ...
+
+Please fix and resubmit.
+
+
+Is there some reference about the method you can add in the Description 
+field in the form Authors (year) <doi:.....>?
+
+### Resubmission on April 2nd, 2019 (planned)
+The problematic links have been fixed:   
+* two were non-canonical CRAN links and   
+* one was a link to an upcoming gallery exhibit that does not yet exist.   
```

Comment:
The link in NEWS.md is a link to a vignette that is getting created as a part of this release.